### ISSUE TITLE
Can't use numeric characters in autocomplete type

### ISSUE THREAD

**Issue by SebastienBelmon** [enquirer/enquirer#112]
Hello,

I'm having an issue which I suppose is a bug. I can't use digit characters in autocomplete type.

### minimal example

```
const test = await prompt({
    type: 'autocomplete',
    name: 'test',
    message: 'Which picture do you want to use ?',
    choices: [
      '100IPLEX_00829.JPG',
      '100IPLEX_00830.JPG',
    ]
  });
  console.log(test);
```

### Expecting behaviour

For example, I would like the user able to enter `830` as input, and the `autocomplete` type prompt to propose `100IPLEX_00830.JPG` to him. But it seems the program does not allow user to use numeric characters while choices are actually strings (as required).

Thank you for your help, i'll try to investigate too. And thank you for this awesome package :smile:

**Comment by jonschlinkert** [comment]
I actually noticed this myself and fixed it locally already. I'll get a fix pushed up ASAP. 

thanks for reporting!

**Comment by ganobrega** [comment]
When will release the fix?

**Comment by ericksepulveda** [comment]
I'm also experiencing this bug/feature. Hopefully you can introduce a flag to turn off the use of numbers to jump to an option.

**Comment by whitecolor** [comment]
@jonschlinkert

This is a really strange bug) just noticed, hope for a fix.

**Comment by doowb** [comment]
The default functionality for numbers is to select a choice. That default can be overridden by passing a `number` function on the options. Using the example above, the options would look like this:

```js
const test = await prompt({
  type: 'autocomplete',
  name: 'test',
  message: 'Which picture do you want to use ?',
  choices: [
    '100IPLEX_00829.JPG',
    '100IPLEX_00830.JPG'
  ],
  number(num) {
    return this.append(num);
  }
});
console.log(test);
```

See [my comment](https://github.com/enquirer/enquirer/pull/259#issuecomment-579934097) for more details.

**Comment by jonschlinkert** [comment]
> The default functionality for numbers is to select a choice.

That's a bug on this prompt though. I thought this was fixed a while ago, wasn't it?

**Comment by doowb** [comment]
I'm not finding any place where it was fixed on the master or 3.0 branches.

If the fix is just adding the `number` method to `Autocomplete` and updating the tests, I'm all for that.

**Comment by jonschlinkert** [comment]
> If the fix is just adding the `number` method to `Autocomplete` and updating the tests, I'm all for that.

Yeah I think that's all it was. It might have been on the dev branch. Or maybe I'm hallucinating.
